README.md

```diff
@@ -56,7 +56,7 @@ The add-in provides the ability to create geodesic features such as lines, circl
 * Running the add-in
 	* To run from a stand-alone deployment
 		* ArcMap
-			* Install the add-in from the application folder by double clicking the **ArcMapAddinDistanceAndDirection.esriAddIn** file.
+			* Install the add-in from the latest Release on Github (https://github.com/Esri/distance-direction-addin-dotnet/releases)
```

Comment:
If you have removed the application folder from GitHub how will this file get into the release? As a separate file/attachment?

Speaking of releases you might want to create a (pre-) release if this is included with the beta - so you can more easily trace back to that version.